addons/l10n_id/data/account_tax_template_data.xml

```diff
@@ -160,13 +160,13 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a_2_122102'),
+                'account_id': ref('l10n_id_21221020'),
             }),
         ]"/>
     </record>
 
-    <record id="tax_PT2" model="account.tax.template">
-        <field name="description">PT2</field>
+    <record id="tax_PT0" model="account.tax.template">
+        <field name="description">PT0</field>
```

Comment:
Why ?

Reply:
![image](https://user-images.githubusercontent.com/82074749/164592430-728213bf-1ca2-4a94-99f2-da823c1266e7.png)
Image attached is example from the old localization

A lot of clients find it annoying to see unsymmetric labeling and name:
(Previous l10n_id taxes)
ST1 => 10% Sales || PT1 => 10% Purchase  [this one is fine]
ST0 => 0% Sales   || PT0 => Exempt Purchase [not matching]
ST2 => Exempt Sales || PT2 => 0% Purchase  [not matching]